### PR TITLE
add basic skeleton of sign-in, home and all articles pages

### PR DESCRIPTION
**Pull request by nwilson7**

### CODE REVIEW

File: src/components/Signin.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function Signin() {
+  return (
+    <form>
+      <label>Email: </label>
+      <input type="text" />
+      <button type="submit">Log in</button>
+    </form>
+  );
+}
```
Comment (nc-rayhaan) [inline src/components/Signin.jsx:9]: Please remove this for now. I get the idea behind it, but you won' be using email to log in. Took me a while to realise how to get to your articles!

File: src/App.jsx [hunk 2]
```diff
@@ -1,8 +1,33 @@
...
+import { getCategories } from "./api";
 
 function App() {
-  return <></>;
+  const [category, setCategory] = useState([]);
+  useEffect(() => {
+    getCategories().then(({ data }) => {
+      setCategory(data);
+    });
+  }, []);
```
Comment (nc-rayhaan) [inline src/App.jsx:17]: You aren't doing anything with this state currently. Please get rid of it until you need it

File: src/App.jsx [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+  return (
+    <>
+      <Header />
+      <Routes>
+        <Route path="/" element={<Signin />} />
+        <Route
+          path="/home"
+          element={<Home category={category} setCategory={setCategory} />}
```
Comment (nc-rayhaan) [inline src/App.jsx:25]: Same with the Home component. Please get rid until you have it working 😄

**Review by nc-rayhaan — APPROVED** [review]
Overall, great work! I've left some comments for you. Please do fix these and then merge it in 😄

File: src/components/Signin.jsx (hunk 1 above)
Comment (nwilson7) [reply src/components/Signin.jsx:9]: Remove the entire component?  Or just the email label for now?

File: src/App.jsx (hunk 3 above)
Comment (nwilson7) [reply src/App.jsx:25]: I'm actually using the Home component to generate my navbar but the Nav should probably be somewhere else as I need it on every page except the sign-in page

**Pull request merged**